### PR TITLE
[WIP] Enhancement for Yarn & NPM colors

### PR DESCRIPTION
**Pull request by cgmx**
## This is a purely stylistic change.

This adds a simple output color when installing dependency's.
It uses the Yarn and NPM brand colors by leveraging `chalk.hex(<hex color>)(<string>)` and says which package manager it's using, either NPM or Yarn

Examples:
*With Yarn*
screen shot 2017-11-07 at 12 22 17 pm

*With NPM*
screen shot 2017-11-07 at 12 24 13 pm

I also added the color to the `To get started` commands (for Yarn and NPM respectively):

screen shot 2017-11-07 at 12 26 50 pm
screen shot 2017-11-07 at 12 28 04 pm

This adds no functionality but is a *cute* stylistic change.

### CODE REVIEW

File: src/commands/create.js [hunk 1]
```diff
@@ -53,7 +53,13 @@ export default async () => {
 
   const isYarn = shouldUseYarn()
 
-  console.log('=> Installing dependencies...')
+  /**
+   * Corbin Matschull (cgmx) - basedjux@gmail.com
+   *
+   * Deps install output will now show WHAT dependency manager-
+   * - you're using before the install steps.
+   */
```
Comment (tannerlinsley) [inline src/commands/create.js]: Let's refrain from these types of comments in the code unless it's to explain complex logic.

Comment (cgmx) [reply src/commands/create.js]: Not a problem. Keep comments to complex logic. 👍

**Comment by cgmx** [comment]
Removed that comment block.

File: src/commands/create.js [hunk 2]
```diff
@@ -68,9 +68,9 @@ ${chalk.green('=> To get started:')}
 
     cd ${answers.name}
 
-    ${isYarn ? 'yarn' : 'npm run'} start ${chalk.green('- Start the development server')}
-    ${isYarn ? 'yarn' : 'npm run'} build ${chalk.green('- Build for production')}
-    ${isYarn ? 'yarn' : 'npm run'} serve ${chalk.green('- Test a production build locally')}
+    ${isYarn ? chalk.hex('#2c8ebb')('yarn') : chalk.hex('#CB3837')('npm run')} start ${chalk.green('- Start the development server')}
+    ${isYarn ? chalk.hex('#2c8ebb')('yarn') : chalk.hex('#CB3837')('npm run')} build ${chalk.green('- Build for production')}
+    ${isYarn ? chalk.hex('#2c8ebb')('yarn') : chalk.hex('#CB3837')('npm run')} serve ${chalk.green('- Test a production build locally')}
```
Comment (tannerlinsley) [inline src/commands/create.js]: We should really extract these colors out into the utils or something.  
eg.
```
// Utils.js
export const Colors = {
  yarn: '...',
  npm: '...'
}
```

Heck, we should even Proxy our calls to `chalk` through our own `Chalk.js` file, so we can just use `Chalk.yarn(...)`

Comment (cgmx) [reply src/commands/create.js]: That could actually be a lot cleaner method of making Chalk calls 🤔. 
I'll work on splitting that out.

**Comment by cgmx** [comment]
I split the colors into an object in `utils.js` for now until I finish the Proxy.

**Comment by tannerlinsley** [comment]
Awesome

**Comment by tannerlinsley** [comment]
I can merge this as soon as it's fixed up with master updates :)

**Pull request merged**